text/0025-purchase-order.md

```diff
@@ -551,8 +552,9 @@ message PurchaseOrderPayload {
 }
 
 message CreatePurchaseOrderPayload {
```

Comment:
Doesn't this message also need an optional `CreateVersionPayload`?

Reply:
Ah yeah I was not originally going to make that change here, but I'll add it.